app/app.go

```diff
@@ -150,7 +150,7 @@ const (
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
 	// See https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES-BUILT-IN.md
-	wasmCapabilities = "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2"
+	wasmCapabilities = "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2, babylon"
```

Comment:
```suggestion
	wasmCapabilities = "iterator,staking,stargate,cosmwasm_1_1,cosmwasm_1_2,babylon"
```

Not sure if space is allowed here though

Reply:
tbh everywhere I looked capabilities were defined without whitespaces, so I would leave it like that 😅

Reply:
But here there's a whitespace, no? 😄

Reply:
aah I missed what is diff and what if proposed change 🤦